test cases/cuda/10 cuda dependency/c/meson.build

```diff
@@ -0,0 +1 @@
+exe = executable('prog', 'prog.c', dependencies: dependency('cuda'))
```

Comment:
There should be a `test()` that runs this executable to ensure that the binaries not only compile, but also work correctly when run. The same goes for all other test executables.

Reply:
Done.